x-pack/plugins/osquery/server/routes/fleet_wrapper/get_package_policies.ts

```diff
@@ -29,6 +29,7 @@ export const getPackagePoliciesRoute = (router: IRouter, osqueryContext: Osquery
       const packagePolicyService = osqueryContext.service.getPackagePolicyService();
       const policies = await packagePolicyService?.list(internalSavedObjectsClient, {
         kuery,
+        perPage: 1000,
```

Comment:
ok, I have to ask the devilish question.

What if a very eccentric user created 2k policies? 5k?

Reply:
not every _user_ is a _customer_ and may not be into contacting us, I guess